packages/calcite-components/src/components/combobox/combobox.scss

```diff
@@ -3,10 +3,46 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-combobox-input-height: Specifies the height of the component's input.
+ * @prop --calcite-combobox-border: defines the border around the combobox input and dropdown. References the --calcite-combobox-border-color by default.
```

Comment:
Do we want to have a general border-color prop here? I think it's unlikely an override to the combobox input would also be desired on the dropdown. Maybe just separate combobox-border and combobox-dropdown-border?

As this is a form element I think we'd want to support border-color, shadow, and corner-radius overrides that default to current border color, shadow-none, and sharp.